zerver/lib/push_notifications.py

```diff
@@ -449,9 +450,20 @@ def get_apns_payload(message: Message) -> Dict[str, Any]:
         'custom': {
             'zulip': {
                 'message_ids': [message.id],
+                'sender_email': message.sender.email,
+                'sender_id': message.sender.id
             }
         }
-    }
+    }  # type: Dict[str, Any]
+
+    if message.recipient.type == Recipient.STREAM:
+        apns_data['custom']['zulip']['recipient_type'] = "stream"
+        apns_data['custom']['zulip']['stream'] = get_display_recipient(message.recipient)
+        apns_data['custom']['zulip']['topic'] = message.subject
+    elif message.recipient.type in (Recipient.HUDDLE, Recipient.PERSONAL):
+        apns_data['custom']['zulip']['recipient_type'] = "private"
```

Comment:
I think this block could be made a little easier on the eyes, and more parallel to the code just above it, with `dict.update`:

```
zulip_data = apns_data['custom']['zulip']
if message.recipient.type == Recipient.STREAM:
    zulip_data.update({
        'recipient_type': 'stream',
        'stream': get_display_recipient(message.recipient),
        'topic': message.subject,
    })
elif message.recipient.type in (Recipient.HUDDLE, Recipient.PERSONAL):
    zulip_data['recipient_type'] = 'private'
```

Reply:
Cool, looks much better, will updated 👍